tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -83,5 +84,77 @@ private string TranslateError (string culture, string errorCode)
 			PropertyInfo propertyInfo = typeof (MSBStrings).GetProperty (errorCode);
 			return (string) propertyInfo.GetValue (null, null);
 		}
+
+		[TestCase ("cs-CZ")]
+		[TestCase ("de-DE")]
+		[TestCase ("es-ES")]
+		[TestCase ("fr-FR")]
+		[TestCase ("it-IT")]
+		[TestCase ("ja-JP")]
+		[TestCase ("ko-KR")]
+		[TestCase ("pl-PL")]
+		[TestCase ("pt-BR")]
+		[TestCase ("ru-RU")]
+		[TestCase ("tr-TR")]
+		[TestCase ("zh-CN")]
+		[TestCase ("zh-TW")]
+		public void AllErrorTranslation (string culture)
+		{
+			string errorCode = string.Empty;
+			string errorList = string.Empty;
+			CultureInfo originalCulture = Thread.CurrentThread.CurrentUICulture;
+			string path = $"{Directory.GetCurrentDirectory ()}/TaskTests/LocalizationIgnore";
+
+			List<string> commonIgnoreList = ReadFile ($"{path}/common-Translations.ignore");
+			List<string> cultureIgnoreList = ReadFile ($"{path}/{culture}-Translations.ignore");
+
+			foreach (var errorCodeInfo in typeof (MSBStrings).GetProperties ()) {
+				try {
+					errorCode = errorCodeInfo.Name.ToString ();
+					if (errorCode == "ResourceManager" || errorCode == "Culture")
+						continue;
+
+					Assert.IsFalse (string.IsNullOrEmpty (errorCode), "Error code is null or empty");
+					string englishError = TranslateError ("en-US", errorCode);
+					string newCultureError = TranslateError (culture, errorCode);
+
+					if (commonIgnoreList.Contains (errorCode)) {
+						if (englishError != newCultureError)
+							Assert.IsFalse (true, $"{errorCode} is translated. Remove {errorCode} from ~/LocalizationIgnore/common-Translations.ignore");
```

Comment:
I think what you're looking for is `Assert.Fail`

Reply:
Or `Assert.AreNotEqual (englishError, newCultureError, $"...")` - the advantage of this being that both `englishError` and `newCultureError` will show up in the logs.